woudc_data_registry/models.py

```diff
@@ -588,6 +608,8 @@ def __init__(self, dict_):
         self.name = dict_['name']
 
         self.generate_ids()
+        if 'station_name_id' in dict_.keys():
+            self.contributor_id = dict_['station_name_id']
```

Comment:
```suggestion
            self.station_name_id = dict_['station_name_id']
```

Correction? No `contributor_id` in `StationName`